com.unity.ml-agents/Runtime/Academy.cs

```diff
@@ -45,7 +45,7 @@ void FixedUpdate()
     ///
     /// At initialization, the Academy attempts to connect to the Python training process through
     /// the external communicator. If successful, the training process can train <see cref="Agent"/>
-    /// instances. When you set an agent's <see cref="BehaviorParameters.behaviorType"/> setting
+    /// instances. When you set an agent's <see cref="BehaviorParameters.BehaviorType"/> setting
```

Comment:
Missed some of these in the rename, and the package validation didn't catch them.